app/controllers/goodsProfile/GoodsRecordsSearchResultController.scala

```diff
@@ -107,10 +104,9 @@ class GoodsRecordsSearchResultController @Inject() (
                         )
                     )
                   )
-
               }
           }
-        case None             => Future(navigator.journeyRecovery())
+        case None             => Future.successful(navigator.journeyRecovery())
       }
     }
 
```

Comment:
again a simplification of the code so it reads easier but the functionality has not changed